convert_state.py

```diff
@@ -1,60 +1,65 @@
+    or converting "California" to "CA".
+"""
+import json
 
-
-
-import json  # import json for json processing
 def main():
+    """
+    Read data from JSON file to start main method
+    """
     with open('data.json','r') as file:
         data = file.read()
         data = json.loads(data)
-
-    print(data)
     
-    # Create a dictionary of states -> abbreviations, and a dictionary of abbreviations -> states 
-    stateAbbr = data #dictionary of state abbreviations keys and state name values 
-    stateAbbr2 = {}# dictionary of state name keys and state abbreviation values
-    # state key is the key in the state dictionary. state value is the value in the dictionary 
-    for statekey, statevalue in stateAbbr.items() :
-        stateAbbr2[statevalue] =  statekey
-
+    state_abbreviations = data #dictionary of state abbreviations keys and state name values 
+    state_names = {} # dictionary of state name keys and state abbreviation values
 
+    # state key is the key in the state dictionary. state value is the value in the dictionary 
+    for state_key, state_value in state_abbreviations.items():
+        state_names[state_value] = state_key
 
-    #loop
     while True:
         print('1. Convert state to abbreviation')
-        print("2. Convery abreviasion to state")
-        print(' 3. quit')
-        choice =   input("Enter choice : ")
-
-        if choice=="1":
-
-            convertStateToAbbreviation(stateAbbr2)
-        elif  choice == "2":
-            convert_abbreviation_to_state(stateAbbr)
-        
-        
-        elif choice==  "3":
+        print('2. Convert abbreviation to state')
+        print('3. Quit')
+        choice = input('Enter choice: ')
+
+        if choice =='1':
+            convert_state_to_abbreviation(state_names)
+        elif choice == '2':
+            convert_abbreviation_to_state(state_abbreviations)
+        elif choice ==  '3':
             break
         else:
-            print('try again')
-def convertStateToAbbreviation(dictionary ):
-    userInput   = input("Enter state name").  capitalize()
-    result = dictionary.get(userInput  )
-    if result == None:
-        print(' state not found')
-    else:
-        print ( "The abbreviation for " +  userInput+" is " + result)
-def convert_abbreviation_to_state(dictionary):
-    userInput   = input("Enter abbrviation name").upper()
-    result = dictionary.get(userInput)
-    if result == None:
-        print(' abreviation not found')
-    else:
-        print("the state with the abbreviation   " + userInput + " is " + result)
+            print('try again.')
 
 
+def convert_state_to_abbreviation(dictionary):
+    """
+    Converts user input for full state name into state abbrevation
+    """
+    user_input = input('Enter state name: ').capitalize()
+    result = dictionary.get(user_input)
+    if result is None:
+        print('state not found.')
+    else:
+        print ( 'The abbreviation for ' +  user_input + 'is' + result)
```

Comment:
Recommend f-string instead of concatenation

Reply:
@claraj f-string added